tests/utils.py

```diff
@@ -135,6 +136,30 @@ def running_on_jenkins() -> bool:
     return RUNNING_ON_JENKINS
 
 
+def running_in_multi_threaded_mode() -> bool:
+    """Whether tests are currently running in multi-threaded mode."""
```

Comment:
can we do it like how cte, sql simplifier is triggered , instead of checking environment variable

Reply:
Yep. I'll make that update.